docs/source/index.rst

```diff
@@ -1,81 +1,40 @@
 |PyPI| |Bioconda| |Downloads| |CI| |Notebooks| |Docs| |Codecov|
 
-CellRank for directed single-cell fate mapping
-==============================================
+CellRank: dynamics from multi-view single-cell data
+===================================================
 
-.. image:: https://raw.githubusercontent.com/theislab/cellrank/master/resources/images/cellrank_fate_map.png
+.. image:: https://raw.githubusercontent.com/theislab/cellrank/master/docs/source/_static/img/cellrank_overview.png
    :width: 600px
    :align: center
 
-.. sidebar:: Key Contributors
+**CellRank** :cite:`lange:22` is a modular framework to study cellular dynamics based on Markov state modeling of
+multi-view single-cell data. It estimates differentiation direction based on a varied (and growing!) number of
+biological priors including RNA velocity, pseudotime, developmental potential and experimental time points.
+See :doc:`about CellRank <about/index>` to learn more.
 
-    * `Marius Lange`_: lead developer, initial CellRank conception, maintainer
-    * `Michal Klein`_: senior developer, design & architecture, maintainer
-    * `Philipp Weiler`_: developer
+CellRank scales to large cell numbers, is fully compatible with the `scverse`_ ecosystem, and is easy to use. In the
+backend, it is powered by the `pyGPCCA package <https://pygpcca.readthedocs.io/>`_ :cite:`reuter:19,reuter:22`. Feel
```

Comment:
@Marius1311, you are still pointing to RTD here. I guess pointing to GitHub (https://github.com/msmdev/pyGPCCA) here, might be more appropriate?

Reply:
done.